doc/source/train/getting-started-pytorch-lightning.rst

```diff
@@ -27,11 +27,13 @@ For reference, the final code is as follows:
         # Your PyTorch Lightning training code here.
 
     scaling_config = ScalingConfig(num_workers=2, use_gpu=True)
+    # For other types of accelerators such as HPUs.
+    # scaling_config = ScalingConfig(num_workers=2, resources_per_worker={"HPU": 1})
```

Comment:
Similarly, we may not want to add example or specifically mention HPUs in the ML framework quick start docs. Please constraint HPU specific words and example within the `using-gpu.rst`. You may change `CPU or GPU devices` into `CPU, GPU or other accelerator devices` and add a hyperlink from the other accelerator devices to the `using-gpus` page.